rst/cartridge_admin.rst

```diff
@@ -718,11 +847,28 @@ Connecting to nodes via CLI
 Each Tarantool node (``router``/``storage``) provides an administrative console
 (Command Line Interface) for debugging, monitoring, and troubleshooting. The
 console acts as a Lua interpreter and displays the result in the human-readable
-YAML format. To connect to a Tarantool instance via the console, say:
+YAML format.
 
-..  code-block:: bash
+To connect to a Tarantool instance via the console, you can choose
+one of the commands:
+
+#.  Old-fashioned way:
```

Comment:
```suggestion
*   Old-fashioned way:
```

Reply:
This should be an unordered list (bullet list).
Numbered lists show sequential steps.